benchmark.go

```diff
@@ -39,8 +40,35 @@ import (
 var (
 	errNegativeDuration = errors.New("duration cannot be negative")
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
+
+	// Global quantiles array for obtaining latency values from getLatencies() method
```

Comment:
since this is a global, we can be more explicit on why we are doing this. Looks like this is mainly so we don't create the same array in test, so along the lines of "using a global quantiles array mainly for ease of testing, and not passing the same arpary around to multiple functions".

also, a nit, we usually prefix global with `_` to infer it's package local: https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_